### PR TITLE
Add notion of default ACL, every controlled resource must have ACL

### LINKED ISSUE #163: Reconcile WAC and the possibility of uncontained resources in Fedora
The Web Access Control spec relies upon [all documents being contained](https://github.com/solid/web-access-control-spec#containers-and-inherited-acls) and [every root container must have an ACL resource](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm). The Fedora API spec allows for uncontained resources which creates situations where the WAC rules don't work. Options I can see include:

  * Say that every uncontained resource (that isn't itself an ACL) MUST have an ACL resource (implication for DELETE debate: if a DELETE zapped a parent the server would need to propagate ACL down to orphans)
  * Say that at the end of the WAC inheritance process some default should be used if no ACL was found (question about how that default is configured or set/adjusted)

### PR DESCRIPTION
**Pull request by zimeon**
Fixes #163

### CODE REVIEW

**Comment by awoods** [comment]
Minor Travis issue with line length.

**Comment by zimeon** [comment]
Thanks @awoods - was in process of fixing, and now fixed

File: index.html [hunk 1]
```diff
@@ -829,24 +829,29 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
...
       </section>
       <section id="link-rel-acl">
         <h2>ACLs are discoverable via Link Headers</h2>
         <p>
-          A conforming server MUST advertise the linked Access Control List Resource (ACL) for a controlled resource in
-          HTTP responses with a <code>Link: rel="acl"</code> header. This interaction pattern is described in [[!WEBAC]]
-          and [[!SOLIDWEBAC]].
+          A conforming server MUST advertise the appropriate Access Control List Resource (ACL) for every controlled
+          resource in HTTP responses with a <code>Link: rel="acl"</code> header. This interaction pattern is described
+          in [[!SOLIDWEBAC]].
```
Comment (awoods) [inline index.html]: Recommend adding a direct link to: https://github.com/solid/web-access-control-spec#acl-resource-location-discovery

**Review by awoods — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (zimeon) [reply index.html]: Good call, will update PR

File: index.html [hunk 2]
```diff
@@ -829,24 +829,31 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
       <section id="solid-ldp-acls">
         <h2>ACLs are LDP RDF Sources</h2>
         <p>
-          The linked Access Control List Resource (ACL) for a controlled resource by a conforming server MUST itself be
+          The Access Control List Resource (ACL) for a controlled resource on conforming server MUST itself be
```
Comment (escowles) [inline index.html]: Minor edit: can you change "on conforming server" to "on a conforming server"?

**Review by escowles — APPROVED** [review]
👍  Other than a very minor edit, this looks good to me.

File: index.html (hunk 2 above)
Comment (zimeon) [reply index.html]: Thanks @escowles, will update

**Review by awoods — APPROVED** [review]

**Comment by acoburn** [comment]
One question about the default ACL and the corresponding link header. If a server `MUST` supply a default ACL for controlled resources that don't have an ACL defined in the relevant hierarchy, if follows that the server `MUST` also provide a link to that ACL in the HTTP response headers. Does that imply that the default ACL must be dereferencable?

**Comment by zimeon** [comment]
@acoburn - yes, I think so. I wonder whether we should add that explicitly (it is testable)?

**Comment by acoburn** [comment]
@zimeon thanks! (n.b. this is another difference w/ the current Fedora WebAC module, but that's entirely OK from my perspective).

**Comment by zimeon** [comment]
Have created #175 to separate the decision about saying default MUST be resolvable

**Review by dannylamb — APPROVED** [review]

**Review by barmintor — APPROVED** [review]

**Pull request merged**